packages/test/test-utils/src/containerUtils.ts

```diff
@@ -46,3 +48,16 @@ export async function waitForContainerConnection(
 			: timeoutPromise(executor, timeoutOptions);
 	}
 }
+
+/**
```

Comment:
New utility function `getContainerEntryPointBackCompat`

Reply:
Maybe add a note of when it can be removed and create a work item to do that? It can be removed when LTS is past this version right?